src/main/java/org/sagebionetworks/web/client/widget/entity/registration/WidgetRegistrarImpl.java

```diff
@@ -261,7 +265,11 @@ public WidgetRendererPresenter getWidgetRendererForWidgetDescriptorAfterLazyLoad
 		} else if (contentTypeKey.equals(REFERENCE_CONTENT_TYPE)) {
 			presenter = ginInjector.getReferenceRenderer();
 		} else if (contentTypeKey.equals(PROVENANCE_CONTENT_TYPE)) {
-			presenter = ginInjector.getProvenanceRenderer();
+			if (DisplayUtils.isInTestWebsite(ginInjector.getCookieProvider())) {
```

Comment:
Please add to https://sagebionetworks.jira.com/wiki/spaces/SWC/pages/2608988161/SWC+Experimental+Mode+Features